release-please-config.json

```diff
@@ -0,0 +1,47 @@
+{
+  "bootstrap-sha": "753e989de7b7663a389379fd1dfdaaba3d920c2e",
```

Comment:
Currently this points here: https://github.com/open-telemetry/opentelemetry-js-contrib/commit/753e989de7b7663a389379fd1dfdaaba3d920c2e

But will need to be updated when there's a squash & merge.

Reply:
Just done